website/content/docs/api-gateway/usage/route-to-peered-services.mdx

```diff
@@ -0,0 +1,77 @@
+---
+page_title: Route Traffic to Peered Services
+description: Learn how to configure Consul API Gateway to route traffic to services connected to the mesh through a peering connection.
+---
+
+# Route Traffic to Peered Services
+
+This topic describes how to configure Consul API Gateway to route traffic to services connected to the mesh through a peering connection.
+
+## Requirements
```

Comment:
TODO: Requires Consul 1.14

Reply:
Also need to update the matrix saying that we don't support WAN federation today